src/services/clients/dissolutionApi.client.ts

```diff
@@ -30,4 +33,24 @@ export class DissolutionApiClient {
     )
     return response.data
   }
+
+  public async getDissolution(token: string, companyNumber: string): Promise<Optional<DissolutionGetResponse>> {
+    const response: Optional<AxiosResponse<DissolutionGetResponse>> = await this.axios.get(
+      `${this.DISSOLUTIONS_API_URL}/dissolution-request/${companyNumber}`,
+      {
+        headers: {
+          Authorization: 'Bearer ' + token,
+          'Content-Type': 'application/json',
+          'Accept': 'application/json',
+        }
+      }
+    ).catch((err: AxiosError) => {
```

Comment:
Using async await you should never need to do `.catch`. Instead do:
```
try {
  const response:AxiosResponse<DissolutionGetResponse> = ...
  return response.data
} catch (err) {
  if (err.response!.status === 404) {
    return null
  }

  return Promise.reject(err)
}
```

Also, inside an async function always use `Promise.reject` instead of `throw`